tests/test_e2e_10_mef_eline.py

```diff
@@ -1369,3 +1370,588 @@ def test_155_removing_evc_metadata_persistent(self):
+            "uni_a": {
+                "tag": {"tag_type": 1, "value": 100},
+                "interface_id": "00:00:00:00:00:00:00:01:1"
+            },
+            "uni_z": {
+                "tag": {"tag_type": 1, "value": "untagged"},
+                "interface_id": "00:00:00:00:00:00:00:02:1"
+            }
+        }
+        api_url = KYTOS_API + '/mef_eline/v2/evc/'
+        response = requests.post(api_url, data=json.dumps(payload), headers={'Content-type': 'application/json'})
+        assert response.status_code == 201, response.text
+        data = response.json()
+        assert 'circuit_id' in data
+        time.sleep(10)
+
+        expected = [
+            {"match": {"in_port": 1, "dl_vlan": 100},
+            "actions": [
+                {"action_type": "pop_vlan"},
+                {"action_type": "push_vlan", "tag_type": "s"},
+                {"action_type": "set_vlan", "vlan_id": 1},
+                {"action_type": "output", "port": 3}
+            ],
+            "priority": 20000},
+            {"match": {"in_port": 1, "dl_vlan": 0},
+            "actions": [
+                {"action_type": "push_vlan", "tag_type": "c"},
+                {"action_type": "set_vlan", "vlan_id": 100},
+                {"action_type": "push_vlan", "tag_type": "s"},
+                {"action_type": "set_vlan", "vlan_id": 1},
+                {"action_type": "output", "port": 2}
+            ],
+            "priority": 20000}
+        ]
+
+        api_url = KYTOS_API + '/flow_manager/v2/flows/00:00:00:00:00:00:00:01'
+        response = requests.get(api_url)
+        data = response.json()["00:00:00:00:00:00:00:01"]["flows"]
+        common_flow = self.get_flow_by_vlan_match(data, 100)
+        assert common_flow["match"] == expected[0]["match"]
+        assert common_flow["priority"] == expected[0]["priority"]
+        assert common_flow["instructions"][0]["actions"] == expected[0]["actions"]
+
+        api_url = KYTOS_API + '/flow_manager/v2/flows/00:00:00:00:00:00:00:02'
+        response = requests.get(api_url)
+        data = response.json()["00:00:00:00:00:00:00:02"]["flows"]
+        untagged_flow = self.get_flow_by_vlan_match(data, 0)
+        assert untagged_flow["match"] == expected[1]["match"]
+        assert untagged_flow["priority"] == expected[1]["priority"]
+        assert untagged_flow["instructions"][0]["actions"] == expected[1]["actions"]
+
+        h11, h2 = self.net.net.get('h11', 'h2')
+        h11.cmd('ip link add link %s name vlan100 type vlan id 100' % (h11.intfNames()[0]))
+        h11.cmd('ip link set up vlan100')
+        h11.cmd('ip addr add 100.0.0.11/24 dev vlan100')
+        h2.cmd('ip addr add 100.0.0.2/24 dev %s' % (h2.intfNames()[0]))
+        result = h11.cmd('ping -c1 100.0.0.2')
+        assert ', 0% packet loss,' in result
+
+        # Clean up
+        h11.cmd('ip link del vlan100')
+        h2.cmd('ip addr del 100.0.0.2/24 dev %s' % (h2.intfNames()[0]))
+        self.net.restart_kytos_clean()
```

Comment:
```suggestion
```

Reply:
Done in commit [89d4f76e090d3feb6275dfb07494de4b37869739](https://github.com/kytos-ng/kytos-end-to-end-tests/pull/213/commits/89d4f76e090d3feb6275dfb07494de4b37869739)